### DISCUSSION TITLE
single delay in preparing the .mp3

### DISCUSSION THREAD

**Discussion by 918483** [savonet/liquidsoap#1756] [Q&A] [answered]
Could someone give a light I recently created a system on time that uses the single but when it looks for the audio it prepares with 2 minutes of delay is there any way for liquidsoap to prepare the audio in real time?

obs: the mp3 file is updated every 1 minute to have the correct time, but liquidsopa prepares it with 2 minutes before
```
single(id="time","/home/streaming/2706/right_time.mp3")
```

**Comment by 918483** [comment]
@toots Can you help me in this part I need the mp3 to be played in real time when requested via single the file contains the same name but it is updated every 1 minute that in the case and the right time I created but when it will play it doesn't get the file that was generated at that time but yes the previous one seems to load it before then stay with the late schedule can you help me?

**Comment by 918483** [comment]
@smimram if you can also help me, thank you!

**Comment by 918483** [comment]
I'm using like this
```
right_time = single(id="time","/home/streaming/2706/right_time.mp3")
time = audio_to_stereo(right_time)
radio = rotate(weights = [1,1],[time,radio])
```

**Comment by toots** [comment, marked as answer]
Hi. I would suggest to look at `input.ffmpeg` for this, I believe this should work in real time.
